<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>KevaAddKeyDialog</class>
    <widget class="QDialog" name="KevaAddKeyDialog">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>800</width>
                <height>400</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string notr="true">Create new record</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
            <item>
                <layout class="QGridLayout" name="gridKeyLableLayout">
                    <item column="1">
                        <widget class="QLabel" name="keyLable">
                            <property name="text">
                                <string>Key</string>
                            </property>
                            <property name="alignment">
                                <set>Qt::AlignLeft</set>
                            </property>
                        </widget>
                    </item>
                    <item column="2">
                        <widget class="QLabel" name="keyCounter">
                            <property name="text">
                                <string/>
                            </property>
                            <property name="alignment">
                                <set>Qt::AlignRight</set>
                            </property>
                        </widget>
                    </item>
                </layout>
            </item>
            <item>
                <widget class="QLineEdit" name="keyText">
                    <property name="toolTip">
                        <string>New key</string>
                    </property>
                </widget>
            </item>
            <item>
                <spacer name="horizontalSpacer">
                    <property name="orientation">
                        <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                        <size>
                            <width>40</width>
                            <height>15</height>
                        </size>
                    </property>
                </spacer>
            </item>
            <item>
                <layout class="QGridLayout" name="gridValueLableLayout">
                    <item column="1">
                        <widget class="QLabel" name="valueLabel">
                            <property name="text">
                                <string>Value</string>
                            </property>
                            <property name="alignment">
                                <set>Qt::AlignLeft</set>
                            </property>
                        </widget>
                    </item>
                    <item column="2">
                        <widget class="QLabel" name="valueCounter">
                            <property name="text">
                                <string/>
                            </property>
                            <property name="alignment">
                                <set>Qt::AlignRight</set>
                            </property>
                        </widget>
                    </item>
                </layout>
            </item>
            <item>
                <widget class="QPlainTextEdit" name="valueText">
                    <property name="toolTip">
                        <string>New value</string>
                    </property>
                </widget>
            </item>
            <item>
                <widget class="QDialogButtonBox" name="buttonBox">
                    <property name="orientation">
                        <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="standardButtons">
                        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
                    </property>
                </widget>
            </item>
        </layout>
    </widget>
    <resources/>
</ui>
